utils/realtek-poe/Makefile

```diff
@@ -0,0 +1,32 @@
+## SPDX-License-Identifier: GPL-2.0-or-later
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=realtek-poe
+PKG_RELEASE:=$(AUTORELEASE)
+PKG_LICENSE:=GPL-2.0
+PKG_MAINTAINER:=Hurricos <hurricos@gmail.com>
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/Hurricos/realtek-poe.git
+PKG_SOURCE_VERSION:=b07fbee998350ef15010eb1e4073c804e8e9217b
+PKG_MIRROR_HASH:=3d88c4c61b696c2b5473b3538b922ef2f8adcb9c3d9f5c34152602c053572714
+CMAKE_SOURCE_SUBDIR:=src
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/realtek-poe
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Realtek PoE Switch Port daemon
+  DEPENDS:=@TARGET_realtek +libubox +libubus +libuci
+endef
+
+define Package/realtek-poe/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/realtek-poe $(1)/usr/bin/
+	$(CP) $(PKG_BUILD_DIR)/files/* $(1)
```

Comment:
Also i think you should use INSTALL_DATA and INSTALL_CONF  https://openwrt.org/docs/guide-developer/packages#file_installation_macros